enacts/wat_bal/maproom_monit.py

```diff
@@ -352,6 +352,8 @@ def wat_bal_ts(
         for wbo in water_balance_outputs:
             if map_choice == "paw" and wbo.name == "sm":
                 ts = 100 * wbo / taw
+            elif map_choice == "water_excess" and wbo.name == "sm":
+                ts = ((wbo / taw) == 1).cumsum()
```

Comment:
Using `==` with floating point numbers is often not the right thing to do.  Is `x / x == 1` guaranteed to be true? When `wbo` gets truncated to `taw`, does it always end up exactly equal to `taw`? Probably, but `isclose` might be safer.